src/components/sign/SignIn.js

```diff
@@ -0,0 +1,76 @@
+import { useFormik } from "formik";
+// import * as Yup from "yup";
+import axios from "axios";
+import { useContext } from "react";
+import MainContext from "../../contexts/MainContext";
+import { logIn } from "../API/APIs";
+import Error from "../../Error";
+
+const SignIn = ({ handleSign, signView }) => {
+    const { err, handleErr, setErr } = useContext(MainContext)
+
+    const formik = useFormik({
+        initialValues: {
+            email: "",
+            password: ""
+        },
+
+        // validationSchema: Yup.object({
+        //     email: Yup.string()
+        //         .required('Required!!'),
+        //     password: Yup.string()
+        //         .required('Required!!')
+        // }),
+
+        onSubmit: () => {
+            setErr(null);
+            const body = formik.values;
+            axios.post(logIn, body)
+                .then((res) => {
+                    handleSign(res.data);
+                    setErr(null);
```

Comment:
there's no need for this line here since we already reset the ```err``` when we execute the ```onSubmit``` function